romancal/datamodels/schemas/reference_files/wfi_img_photom.schema.yaml

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/wfi_img_photom.schema"
+title: WFI imaging photometric flux conversion data model
+allOf:
+- $ref: referencefile.schema
+- type: object
+  properties:
+    phot_table:
+      title: Photometric flux conversion factors table
+      datatype:
+      - name: optical_element
+        datatype: [ascii, 12]
+      - name: photmjsr
```

Comment:
It looks like this is MJy/steradian? Can we add some text to clarify that?

Reply:
I added 2 titles.